### ISSUE TITLE
Support for scanning Java projects

### ISSUE THREAD

**Issue by gsluthra** [Stono/hawkeye#6]
Hi, I would like to have a vulnerability scanner for Java libraries for our project -- something like the OWASP dependency checker: https://www.owasp.org/index.php/OWASP_Dependency_Check

My hope is that we should be able to register our Github java repository with hawkeye, and then it will provide reports on which libraries are vulnerable and should be updated, etc. 

Our project: https://github.com/bahmni

Thanks!

**Comment by nicolastrres** [comment]
hey @Stono, I saw that you were working on this on a different branch, is that still in progress? is there something that we could do to help?

**Comment by Stono** [comment]
Hey @nicolastrres  - thanks for the question.
In summary I got to the point of using the OWASP checker to run the scan but there were a couple of issues that blocked me:

  - The dependency checker requires a project (say, maven) install, which pulls down all the dependencies, rather than looking at dependency files (like the python, node, ruby modules do).  This means it wouldn't work on the SaaS offering (as hawkeye website wouldn't/shouldn't become a build server).  This made me question if the OWASP scanner was the best choice
  - The output from the scanner was quite a complex XML file and I didn't have time to process/digest that into hawkeye results
  - I have limited Java experience 

There were a few others things to that made me just put the brakes on it for a time being.  More than happy for it to be picked up again by someone else, and happy to give input, but being realistic it isn't something that I have time to be investing in right now!
